projects/lib/tsconfig.lib.prod.json

```diff
@@ -0,0 +1,10 @@
+/* To learn more about this file see: https://angular.io/config/tsconfig. */
+{
+  "extends": "./tsconfig.lib.json",
+  "compilerOptions": {
+    "declarationMap": false
+  },
+  "angularCompilerOptions": {
```

Comment:
I would suggest to use this mode in all builds- prod and non prod. What do you think?

Reply:
I just used the angular standard config. I do not know.
What do you think @lazaki

Reply:
if we do not use it than npm link would not work :), as it would compile it with full as default development mode i think